src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ticketRuleForm.tsx

```diff
@@ -27,13 +46,175 @@ class TicketRuleForm extends React.Component<Props, State> {
     });
   };
 
+  onCancel = () => {
+    this.closeModal();
+  };
+
+  getNames = (): string[] => {
+    const names: string[] = [];
+    for (const name in this.props.formFields) {
+      if (this.props.formFields[name].hasOwnProperty('name')) {
+        names.push(this.props.formFields[name].name);
+      }
+    }
+    return names;
+  };
+
+  cleanData = (data: {
+    [key: string]: string;
+  }): {
+    jira_integration?: any;
+    vsts_integration?: any;
+    [key: string]: any;
+  } => {
+    const names: string[] = this.getNames();
+    const formData: {
+      jira_integration?: any;
+      vsts_integration?: any;
+      [key: string]: any;
+    } = {};
+    if (this.props.instance?.hasOwnProperty('jira_integration')) {
+      formData.jira_integration = this.props.instance?.jira_integration;
+    }
+    if (this.props.instance?.hasOwnProperty('vsts_integration')) {
+      formData.vsts_integration = this.props.instance?.vsts_integration;
```

Comment:
@ceorourke not sure what's going on here. Where do we set `jira_integration` and `vsts_integration`?